include/revng/Pipeline/Invokable.h

```diff
@@ -197,12 +207,27 @@ void createAndAppend(std::vector<std::unique_ptr<CLOptionBase>> &Out,
   Out.emplace_back(std::make_unique<Wrapper>(std::forward<ArgsT>(Args)...));
 }
 
+template<typename T, size_t S, typename... ArgsT>
+void createAndAppendPathSwitch(std::vector<std::unique_ptr<CLOptionBase>> &Out,
+                               ArgsT &&...Args) {
+  if constexpr (std::is_same_v<OptionType<T, S>, std::string>) {
+    using Wrapper = CLOptionWrapper<bool>;
+    Out.emplace_back(std::make_unique<Wrapper>(std::forward<ArgsT>(Args)...));
+  }
+}
+
 template<typename T, size_t... S>
 void createCLOption(std::vector<std::unique_ptr<CLOptionBase>> &Out,
                     const std::integer_sequence<size_t, S...> &,
                     llvm::cl::OptionCategory *Cat = nullptr) {
   using cat = llvm::cl::cat;
   (createAndAppend<T, S>(Out, T::Name, getOptionName<T, S>(), cat(*Cat)), ...);
+  (createAndAppendPathSwitch<T, S>(Out,
+                                   T::Name,
+                                   getOptionName<T, S>().str() + "-is-path",
+                                   cat(*Cat),
+                                   llvm::cl::init(false)),
+   ...);
```

Comment:
I understand why you might want this, but I really don't want to have to use two options.

```
revng analyze --ConvertToCABIFunctionType-confidence=/lel --ConvertToCABIFunctionType-confidence-is-path
```

It's not the most ergonomic thing.

I'd go for something like:
```
revng analyze --ConvertToCABIFunctionType-confidence-path=/lel
```

Reply:
Ah! I misunderstood what you meant the other day, I'll fix it.

Reply:
Fixed